config/protractor/protractor-dev.conf.js

```diff
@@ -19,36 +20,44 @@ let config = {
     jasmine.getEnv().addReporter(new SpecReporter());
 
     return new Promise((resolve, reject) => {
-      const url = 'https://github.com/blackbaud/skyux-template';
-      const branch = 'builder-dev';
 
-      common.rimrafPromise(common.tmp)
-        .then(() => common.exec(`git`, [
-          `clone`,
-          `-b`,
-          branch,
-          `--single-branch`,
-          url,
-          common.tmp
-        ]))
-        .then(() => common.exec(`npm`, [`i`, '--only=prod'], common.cwdOpts))
-        .then(() => common.exec(`npm`, [`i`, `../`], common.cwdOpts))
-        .then(resolve)
-        .catch(reject);
-    });
-  },
+      if (fs.existsSync(common.tmp) && !process.argv.includes('--clean')) {
 
-  onComplete: () => {
+        console.log('');
+        console.log('*********');
+        console.log('Running fast e2e tests');
+        console.log(`Delete ${common.tmp} to have the install steps run.`);
+        console.log('*********');
+        console.log('');
 
-    // Catch any rogue servers
-    common.afterAll();
+        resolve();
 
-    return new Promise((resolve, reject) => {
-      common.rimrafPromise(common.tmp)
-        .then(resolve)
-        .catch(reject);
+      } else {
+
+        const url = 'https://github.com/blackbaud/skyux-template';
+        const branch = 'master';
```

Comment:
Changing this wasn't intentional... chalking it up to a bad merge.

Learning about it though, I'm torn.  It seems nice for development for new features in builder that may require a change to the template.  On the other hand, running builder's e2e tests is used to test that `skyux new` runs as intended for consumers, in which case they'd be using the `master` branch.

I'll change it back to `builder-dev` for the sake of this PR and we can talk more about it later.